### ISSUE TITLE
0.5 to 0.6 update fails

### ISSUE THREAD

**Issue by gingerbeardman** [mangerlahn/Latest#47]
Stuck on this:

![Screen shot 2019-09-02 at 10 44 36](https://user-images.githubusercontent.com/49612/64105741-bcd9d580-cd6e-11e9-89af-d97687b5a4a8.png)

**Comment by mangerlahn** [comment]
Ouch, this should not happen. I see what I can do there. Thanks!

**Comment by mangerlahn** [comment]
Wow, I really messed that one up 😅

Somehow I forgot that Sparkle officially does not support Sandboxing, a security mechanism of macOS. I enabled that with 0.5, which completely broke the update system. Oops again. 

I changed the Sparkle implementation with 0.6 completely, so this problem is gone from 0.6 on.

Good news is that I can somewhat fix this by redirecting people to the website to download a new version. I will do that as soon as I have 0.6.1 ready. I am very sorry for the inconvenience here!
